Source/Core/DolphinQt2/Settings/InterfacePane.cpp

```diff
@@ -19,6 +20,56 @@
 
 #include "DolphinQt2/Settings.h"
 
+static QComboBox* MakeLanguageComboBox()
+{
+  constexpr struct
+  {
+    const wchar_t* name;
+    const char* id;
+  } languages[] = {
+      {L"Bahasa Melayu", "ms"},               // Malay
+      {L"Catal\u00E0", "ca"},                 // Catalan
+      {L"\u010Ce\u0161tina", "cs"},           // Czech
+      {L"Dansk", "da"},                       // Danish
+      {L"Deutsch", "de"},                     // German
+      {L"English", "en"},                     // English
+      {L"Espa\u00F1ol", "es"},                // Spanish
+      {L"Fran\u00E7ais", "fr"},               // French
+      {L"Hrvatski", "hr"},                    // Croatian
+      {L"Italiano", "it"},                    // Italian
+      {L"Magyar", "hu"},                      // Hungarian
+      {L"Nederlands", "nl"},                  // Dutch
+      {L"Norsk bokm\u00E5l", "nb"},           // Norwegian // wxWidgets won't accept "no"
```

Comment:
TIL there's two different written forms of Norwegian! Yeah, looks like "nb" is correct regardless of what wxWidgets will accept. I removed this comment from DolphinQt, and added another commit removing it from DolphinWX.